.github/scripts/build.sh

```diff
@@ -73,7 +77,7 @@ if [[ "$PLATFORM" =~ cygwin ]]; then
 fi
 
 if [ "$CCACHE" = 1 ] ; then
-  ccache -svv
+  ccache -s
```

Comment:
Why was this line changed?

Reply:
It was failing. For me it looked like a typo.

Reply:
Should be valid and should give the most verbose output https://ccache.dev/manual/4.10.html

Reply:
why is it "vv"?

Reply:
I have "ccache version 3.7.7" locally in WSL. It does not support -v also.